app/controllers/rooms_controller.rb

```diff
@@ -6,8 +6,15 @@ class RoomsController < ApplicationController
 
   # GET /rooms/:public_id
   def show
-    @jams = @room.jams.order(created_at: :desc).last(20).to_a
-    @current_jam = @jams.shift
+    jams = @room.jams.order(created_at: :desc).last(20).to_a
+    @supporting_files = []
```

Comment:
`uploaded_files` ? `supporting` is kind weird

Reply:
ah, I see what you mean, so there is a latest mix and the files that support it? In that case you 'd want to be careful with last 20, it'd prob work most of the time, but you might consider to do a separate call to get the latest mix independently from the rest (so you get it even if there is a lot of supported files and the mix is not picked up by last 20).